src/main/java/com/suse/saltstack/netapi/config/ClientConfig.java

```diff
@@ -35,6 +35,13 @@
     public static final Key<String> PROXY_PASSWORD = new Key<>();
 
     /**
+     * Maximum websocket message length in characters. The default value corresponds to
+     * 10 MB, assuming that one character takes up 2 bytes. This limit can be disabled
+     * by setting a value <= 0.
+     */
+    public static final Key<Integer> WEBSOCKET_MAX_MESSAGE_LENGTH = new Key<>(5242880);
```

Comment:
Can we use hexadecimals here? I think that `0x500000` looks/understood better. Because  `0x500000 / 0x400 / 0x400` equals 5MB. If 2 bytes per character (UTF16), then the limit is to 2,5MB.